docs/nlp/nlp-mindsdb-hf.mdx

```diff
@@ -182,3 +180,126 @@ On execution, we get:
 ```
 
 For the full library of supported examples please go [here](/nlp/nlp-extended-examples).
+
+### Example using MQL
+
+Let's go through a Sentiment Classification example, but this time we'll use a Mongo database.
+
+<Note>
+    **Using Local Installation of MindsDB**
+
+    Please note that if you use a local installation of MindsDB, instead of MindsDB Cloud, you should install `transformers==4.21.0` to be able to use the Hugging Face models.
+</Note>
+
+We have a sample Mongo database that you can connect to your MindsDB Cloud account by running this command:
+
+```sql
```

Comment:
Do we want to keep the `CREATE DATABASE` syntax and then switch to mongo clients or can we add the 

```
db.databases.insertOne()
```

Reply:
I spoke with Tom that I'll change it to MQL syntax.